docs/release-notes/_snippets/9.1.9/index.md

```diff
@@ -0,0 +1,39 @@
+## 9.1.9 [beats-release-notes-9.1.9]
+
+_This release also includes: [Breaking changes](/release-notes/breaking-changes.md#beats-9.1.9-breaking-changes)._
+
+
+### Features and enhancements [beats-9.1.9-features-enhancements]
+
+
+**Filebeat**
+
+* Log unpublished event count and exit publish loop on input context cancellation. [#47256](https://github.com/elastic/beats/pull/47256) [#47956](https://github.com/elastic/beats/pull/47956) [#48141](https://github.com/elastic/beats/pull/48141) [#48115](https://github.com/elastic/beats/pull/48115) [#47823](https://github.com/elastic/beats/issues/47823)
+
+**Metricbeat**
+
+* Add resource pool id to vsphere cluster metricset. [#47256](https://github.com/elastic/beats/pull/47256) [#47956](https://github.com/elastic/beats/pull/47956) [#48141](https://github.com/elastic/beats/pull/48141) [#48115](https://github.com/elastic/beats/pull/48115) [#47823](https://github.com/elastic/beats/issues/47823)
+
+**Packetbeat**
+
+* Ipfrag2. [#47256](https://github.com/elastic/beats/pull/47256) [#47956](https://github.com/elastic/beats/pull/47956) [#48141](https://github.com/elastic/beats/pull/48141) [#48115](https://github.com/elastic/beats/pull/48115) [#47823](https://github.com/elastic/beats/issues/47823)
```

Comment:
Based on https://github.com/elastic/beats/pull/48090, I think this is what's needed:

```suggestion
* Rework IP fragmentation and make it bound. [#47970](https://github.com/elastic/beats/pull/47970)
```